pkgs/tools/security/passff-host/default.nix

```
@@ -1,27 +1,32 @@
 { stdenv, fetchFromGitHub, python3, pass }:
 
 stdenv.mkDerivation rec {
-  name = "passff-host-${version}";
+  pname = "passff-host";
   version = "1.2.1";
 
   src = fetchFromGitHub {
     owner = "passff";
-    repo = "passff-host";
+    repo = pname;
     rev = version;
     sha256 = "0ydfwvhgnw5c3ydx2gn5d7ys9g7cxlck57vfddpv6ix890v21451";
   };
 
   buildInputs = [ python3 ];
+  makeFlags = [ "VERSION=${version}" ];
 
   patchPhase = ''
     sed -i 's#COMMAND = "pass"#COMMAND = "${pass}/bin/pass"#' src/passff.py
   '';
 
   installPhase = ''
-    install -D bin/testing/passff.py $out/share/passff-host/passff.py
-    cp bin/testing/passff.json $out/share/passff-host/passff.json
-    substituteInPlace $out/share/passff-host/passff.json \
+    substituteInPlace bin/$version/passff.json \
```

This is referencing the environment variable `version` from the nix expression right?

It would be safer to do
```suggestion
    substituteInPlace bin/${version}/passff.json \
```

anywhere else to

---

[Correct](https://github.com/passff/passff/blob/master/Makefile#L5). I tend toward using derivation variables where possible just because interpolation feels dirty and invalidates script data/strings that could otherwise stay constant, but will do - it's not like nagging feelings will make nix actually do magical memoization.